packages/suite-web-landing/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "@trezor/suite-web-landing",
+    "version": "20.10.1",
+    "private": true,
+    "scripts": {
+        "type-check": "tsc --project tsconfig.json",
+        "type-check:watch": "yarn type-check -- --watch",
+        "lint": "eslint . --ext .tsx,.ts",
+        "copy-files": "yarn workspace @trezor/suite-data copy-static-files",
+        "dev": "yarn workspace @trezor/suite-data copy-static-files && next",
+        "build:app": "next build && next export -o build",
+        "build": "run-s copy-files build:app"
+    },
+    "dependencies": {
+        "@trezor/components": "1.0.0",
+        "next": "^9.5.3",
+        "react": "16.13.1",
+        "react-dom": "16.13.1",
+        "react-reveal": "^1.2.2"
```

Comment:
not sure if we shouldn't switch to something supported, but I guess it's not a priority right now 
https://github.com/rnosov/react-reveal/issues/106

Reply:
I think I can switch it to `react-awesome-reveal` which they recommend.

Reply:
Done, thanks